docs/advanced/atomic-swap.md

```diff
@@ -0,0 +1,38 @@
+# Atomic Swaps
+
+---
+
+## Overview
+
+An Atomic Swap is a smart contract technology which makes possible to exchange coins from two different blockchains without having to trust any third party, for example a centralized exchange. 
+For an atomic swap, a cryptocurrency must have the 'OP_CHECKLOCKTIMEVERIFY' opcode, which makes a timelock possible.
```

Comment:
It isn't correct to say this one opcode is necessary - the blog I linked previously explains that there are two possible opcodes which can be used - CLTV or CSV.

And I may be wrong about this but I believe it would be possible to use other mechanisms, so long as they enable restricting access to coins for a specific number of blocks. @jrick could you please confirm?

Reply:
Ok, I edited this section, but not resolving conversation yet.

Reply:
> so long as they enable restricting access to coins for a specific number of blocks

this is exactly what CSV does.